src/pages/scientificServices/pipelines/views/modals/ViewOutputsModal.tsx

```diff
@@ -13,9 +13,46 @@ interface OutputsModalProps {
   onDismiss: () => void;
 }
 
+// const getFileSize = async (url: string): Promise<string> => {
+//   try {
+//     const response = await fetch(url, { method: 'HEAD' });
+//     const size = response.headers.get('content-length');
+//     return size ? formatBytes(parseInt(size)) : 'Unknown size';
+//   } catch {
+//     return 'Unknown size';
+//   }
+// };
+
+// workaround until HEAD requests are allowed by the CORS configuration
+const getFileSize = async (url: string): Promise<string> => {
```

Comment:
this workaround is fine i think? we'd need to add `HEAD` as an allowed method on the CORS configuration for the workspace bucket in order to use the simpler commented-out approach above

Reply:
Just filed https://broadworkbench.atlassian.net/browse/TSPS-585 as a followup